dev/com.ibm.websphere.appserver.features/visibility/public/jaxb-2.3/com.ibm.websphere.appserver.jaxb-2.3.feature

```diff
@@ -0,0 +1,35 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.jaxb-2.3
+visibility=public
+singleton=true
+IBM-App-ForceRestart: uninstall, \
+ install
+IBM-API-Package: \
+  javax.xml.bind;  type="spec", \
+  javax.xml.bind.annotation;  type="spec", \
+  javax.xml.bind.annotation.adapters;  type="spec", \
+  javax.xml.bind.attachment;  type="spec", \
+  javax.xml.bind.helpers;  type="spec", \
+  javax.xml.bind.util;  type="spec"
+IBM-ShortName: jaxb-2.3
+IBM-Process-Types: client, \
+ server
+Subsystem-Name: Java XML Bindings 2.3
+-features=\
+  com.ibm.websphere.appserver.classloading-1.0
+-bundles=\
+  com.ibm.websphere.javaee.activation.1.1; required-osgi-ee:="(&(osgi.ee=JavaSE)(version>=9))"; location:="dev/api/spec/,lib/"; mavenCoordinates="javax.activation:activation:1.1", \
```

Comment:
lets remove the required-osgi-ee attribute here.  It's renamed to something else in integration currently, but also I'd like us to always pull in our copy of the javax.activation API instead of relying on the JDK copy (which is going away in JDK 9+)